.. _exhale_function_namespaceCatch_1ae67297c3e265b0fcd55de371bf408e4e:

Function Catch::throw_domain_error
==================================

- Defined in :ref:`file__home_aakash_mlpack_src_mlpack_tests_catch.hpp`


Function Documentation
----------------------


.. doxygenfunction:: Catch::throw_domain_error(std::string const&)
